<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>game_page</class>
 <widget class="QWidget" name="game_page">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1669</width>
    <height>855</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton{
	background-color: rgb(170, 7, 132);
	color: rgb(0, 0, 0);
	font: 700 25pt &quot;Script MT&quot;;

}</string>
  </property>
  <widget class="QLabel" name="mine_status">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>100</y>
     <width>241</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	background-color: rgb(80, 0, 60);
	color: rgb(255, 255, 255);
	font: 700 20pt &quot;Script MT&quot;;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="competitor_status">
   <property name="geometry">
    <rect>
     <x>1080</x>
     <y>90</y>
     <width>231</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	background-color: rgb(80, 0, 60);
	color: rgb(255, 255, 255);
	font: 700 20pt &quot;Script MT&quot;;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="player_status_1">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>241</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	background-color: rgb(80, 0, 60);
	color: rgb(255, 255, 255);
	font: 700 20pt &quot;Script MT&quot;;
}</string>
   </property>
   <property name="text">
    <string>player status :</string>
   </property>
  </widget>
  <widget class="QLabel" name="player_status_2">
   <property name="geometry">
    <rect>
     <x>1080</x>
     <y>20</y>
     <width>231</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	background-color: rgb(80, 0, 60);
	color: rgb(255, 255, 255);
	font: 700 20pt &quot;Script MT&quot;;
}</string>
   </property>
   <property name="text">
    <string>player status :</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Card_request">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>40</y>
     <width>311</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Card request</string>
   </property>
  </widget>
  <widget class="QPushButton" name="card_2">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>190</y>
     <width>271</width>
     <height>261</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="card_5">
   <property name="geometry">
    <rect>
     <x>1120</x>
     <y>190</y>
     <width>261</width>
     <height>261</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="card_3">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>190</y>
     <width>271</width>
     <height>261</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="card_4">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>480</y>
     <width>271</width>
     <height>241</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="card_6">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>190</y>
     <width>271</width>
     <height>261</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="card_1">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>480</y>
     <width>271</width>
     <height>241</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="card_7">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>490</y>
     <width>261</width>
     <height>231</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
